ledger/streaming/api/proto/streaming_blocks.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+/// Streaming Ledger Sync API.
+
+syntax = "proto3";
+
+package streaming_blocks;
+
+import "blockchain.proto";
+import "external.proto";
+import "quorum_set.proto";
+
+message SubscribeRequest {
+    /// Block index to start from.
+    uint64 starting_height = 1;
+}
+
+message BlockWithQuorumSet {
+    /// The block data.
+    blockchain.ArchiveBlock block = 1;
+    /// QuorumSet when the block was externalized.
+    quorum_set.QuorumSet quorum_set = 2;
+    /// Attestation Verification report.
+    external.VerificationReport report = 3;
+}
+
+message SubscribeResponse {
```

Comment:
On second thought this should probably include the NodeID so that a proper accounting of quorum can be done when doing block verification. The sig is representative, but it makes it potentially difficult to get an accounting of what is going wrong if there are ever issues

Reply:
Why NodeID and not the whole `BlockWithQuorumSet`?

Reply:
Yes, that'll include the members correct, but is it easily identifiable which member the node is in that set in the stream being received without the node id?